bugshaker/src/main/java/com/github/stkent/bugshaker/BugShaker.java

```diff
@@ -216,8 +219,28 @@ public void start() {
 	@Override
 	public void hearShake() {
 		logger.d("Shake detected!");
-		feedbackEmailFlowManager.startFlowIfNeeded(application,
-			false);
+		//if (bugShakerOn){
+			feedbackEmailFlowManager.startFlowIfNeeded(application,
+				false);
+		//}
+	}
+
+	public static void changeStatus(){
+		if (bugShakerOn){
+			bugShakerOn= false;
+		}
+		else if (!(bugShakerOn)){
+			bugShakerOn = true;
+		}
 	}
 
+	public static boolean isBugShakerOn(){
```

Comment:
EBPreferencesFragment calls this to check whether bugshaker is already listening to shakes at that moment